packages/contracts-bridge/contracts/test/utils/BridgeTest.sol

```diff
@@ -25,61 +27,86 @@ import {TypeCasts} from "@nomad-xyz/contracts-core/contracts/XAppConnectionManag
 
 import "forge-std/Test.sol";
 
-contract BridgeTestFixture is Test {
+contract BridgeTestFixture is NomadTest {
     using TypeCasts for bytes32;
     using TypeCasts for address;
     using TypeCasts for address payable;
 
-    // Local variables
-    uint256 mockUpdaterPK;
-    address mockUpdater;
+    // Fixtures
     address bridgeUser;
-    uint32 localDomain;
     uint256 bridgeUserTokenAmount;
+    ERC20Mock localToken;
 
-    // Remote variables
+    // Mock-only state
+    uint256 mockUpdaterPK;
+    address mockUpdater;
     bytes32 remoteBridgeRouter;
-    uint32 remoteDomain;
+    address mockReplica;
```

Comment:
i'm slightly confused when we want to have a `replica` vs. `mockReplica` vs. both (same for home, etc)

Reply:
a mock is a local testing convenience feature, and should not be used during fork tests. given that our test idiom is halfway between mocks and env-independent cheatcode usage, I'm gonna not touch this, as I don't know everywhereit's used.

Reply:
me too. seems code smell, as anna would say